src/ldc/eh/common.d

```diff
@@ -287,12 +287,72 @@ void _d_getLanguageSpecificTables(ubyte* data, ref ubyte* callsite, ref ubyte* a
     debug(EH_personality) printf("  - callsite: %p, action: %p, classinfo_table: %p, ciEncoding: %d\n", callsite, action, classinfo_table, ciEncoding);
 }
 
+// ---------------------------------------------------------------
```

Comment:
Sure. I was mimicking the category headers for the other sections of code, but DDoc standards are more appropriate for this.